src/status_im2/contexts/syncing/syncing_instructions/view.cljs

```diff
@@ -41,11 +42,13 @@
      (i18n/label value)]
 
     :context-tag
-    [quo/context-tag
-     {:override-theme :dark
-      :text-style     {:color colors/white}}
-     (resources/get-mock-image (:source value))
-     (i18n/label (:label value))]))
+
+    [quo.theme/provider {:theme :dark}
```

Comment:
hmm, why are you using the theme provider here? Is there another way to do this? @smohamedjavid ? 🤔 I think you can pass theme to bottom sheet

Reply:
Fixed!